platforms/nuttx/src/px4/common/px4_layer.cmake

```diff
@@ -3,6 +3,7 @@
 add_library(px4_layer
 		${KERNEL_SRCS}
 		cdc_acm_check.cpp
+		SerialImpl.cpp
```

Comment:
Add this also to the [px4_protected_layers.cmake](https://github.com/PX4/PX4-Autopilot/blob/42547a1319d0952d15ef845b8b8560eabf5cb7e5/platforms/nuttx/src/px4/common/px4_protected_layers.cmake#L15)